.github/workflows/release.yml

```diff
@@ -21,10 +21,10 @@ jobs:
           github-token: ${{ secrets.github_token }}
 
       - name: Create Release
-        uses: actions/create-release@v1
+        uses: softprops/action-gh-release@v1
```

Comment:
This appears to be an action that was recommended by Github. Has near-parity wrt the interface.